src/test/ui/proc-macro/invalid-punct-ident-1.stderr

```diff
@@ -1,3 +1,43 @@
+stack backtrace:
```

Comment:
The "stack backtrace" changes happen because you run `x.py test` with `RUST_BACKTRACE=1`.
Reference UI test outputs generally assume that the tests are run without backtraces.